http/1.7/modules/ROOT/pages/http-polling-source-ref.adoc

```diff
@@ -0,0 +1,306 @@
+= HTTP Request Configuration Reference
+:page-aliases: connectors::http/http-polling-source-ref.adoc
+
+The HTTP Polling Source allows you to have an event source that, instead of waiting for an HTTP request from a user (like the HTTP Listener), performs an scheduled HTTP Request to a given HTTP Server.
+
+You can configure the scheduling frequency. May it be a fixed frequency, or a cron expression. You can also configure how to split the HTTP Response from the server, as well how to retrieve watermarking (if any) and ids (if any).
```

Comment:
`split the HTTP Response from the server` -> i think the use case here (process many elements contained in the response) is more important that how it is done (by splitting).